specification/attestation/resource-manager/Microsoft.Attestation/preview/2021-06-01-preview/attestation.json

```diff
@@ -20,7 +20,7 @@
       "type": "oauth2",
       "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
       "flow": "implicit",
-      "description": "Azure Active Directory OAuth2 Flow",
+      "description": "Azure Active Directory OAuth2",
```

Comment:
>Azure Active Directory [](http://example.com/codeflow?start=22&length=22)

you should update the description - [https://learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad)

Reply:
@razvanbadea-msft: This change will not be merged.  It's only being used to test the PR.

@chidozieononiwu: Could you test with a spec like the `contoso` sample, so it's more obvious this is only a test change?